### PR TITLE
feat(guide): Add guide assistance for txn threshold

### PR DESCRIPTION
**Pull request by shruthilayaj**
Add an assistant for per transaction thresholds
in the summary page. Also add the alpha tag.

Screenshots:
Screen Shot 2021-07-07 at 12 18 17 PM

Screen Shot 2021-07-07 at 8 51 59 AM

### CODE REVIEW

File: static/app/components/assistant/getGuidesContent.tsx [hunk 1]
```diff
@@ -308,6 +308,19 @@ export default function getGuidesContent(orgSlug: string | null): GuidesContent
         },
       ],
     },
+    {
+      guide: 'project_transaction_threshold_override',
+      requiredTargets: ['project_transaction_threshold_override'],
+      steps: [
+        {
+          title: t('Response Time Threasholds'),
```
Comment (k-fish) [inline static/app/components/assistant/getGuidesContent.tsx:316]: :smile:

```suggestion
          title: t('Response Time Thresholds'),
```

File: static/app/views/performance/transactionSummary/transactionThresholdModal.tsx [hunk 2]
```diff
@@ -10,6 +10,7 @@ import {ModalRenderProps} from 'app/actionCreators/modal';
 import {Client} from 'app/api';
 import Button from 'app/components/button';
 import ButtonBar from 'app/components/buttonBar';
+import StyledFeatureBadge from 'app/components/featureBadge';
```
Comment (k-fish) [inline static/app/views/performance/transactionSummary/transactionThresholdModal.tsx:13]:
```suggestion
import FeatureBadge from 'app/components/featureBadge';
```

File: static/app/views/performance/transactionSummary/transactionThresholdModal.tsx [hunk 3]
```diff
@@ -235,7 +236,9 @@ class TransactionThresholdModal extends React.Component<Props, State> {
     return (
       <React.Fragment>
         <Header closeButton>
-          <h4>{t('Transaction Settings')}</h4>
+          <h4>
+            {t('Transaction Settings')} <StyledFeatureBadge type="alpha" />
```
Comment (k-fish) [inline static/app/views/performance/transactionSummary/transactionThresholdModal.tsx:240]:
```suggestion
            {t('Transaction Settings')} <FeatureBadge type="alpha" />
```

**Review by k-fish — COMMENTED** [review]
😄

File: static/app/components/assistant/getGuidesContent.tsx (hunk 1 above)
Comment (shruthilayaj) [reply static/app/components/assistant/getGuidesContent.tsx]: good catch haha

**Review by doralchan — APPROVED** [review]

**Pull request merged**